code/modules/mob/living/carbon/human/species/shadekin/shadekin.dm

```diff
@@ -0,0 +1,263 @@
+/datum/species/shadekin
+	name = SPECIES_SHADEKIN
+	name_plural = "Shadekin"
+	blurb = "Waow! A shaadefluffer!"
+	catalogue_data = list(/datum/category_item/catalogue/fauna/shadekin)
+
+	language = LANGUAGE_SHADEKIN
+	assisted_langs = list()
+	unarmed_types = list(/datum/unarmed_attack/claws/shadekin/red)
+	hud_type = /datum/hud_data/shadekin
+	rarity_value = 15	//INTERDIMENSIONAL FLUFFERS
+
+	tail = "shadekintail"
+	tail_animation = 'icons/mob/species/shadekin/tail.dmi'
+	longtail = TRUE
+
+	has_fine_manipulation = 0
+	siemens_coefficient = 0
+	darksight = 10
+
+	slowdown = -0.5
+	item_slowdown_mod = 0.5
+
+	brute_mod = 0.7	// Naturally sturdy.
+	burn_mod = 1.2	// Furry
+
+	warning_low_pressure = 50
+	hazard_low_pressure = -1
+
+	warning_high_pressure = 300
+	hazard_high_pressure = INFINITY
+
+	cold_level_1 = -1	//Immune to cold
+	cold_level_2 = -1
+	cold_level_3 = -1
+
+	heat_level_1 = 850	//Resistant to heat
+	heat_level_2 = 1000
+	heat_level_3 = 1150
+
+	flags =  NO_SCAN | NO_PAIN | NO_SLIP | NO_MINOR_CUT | NO_INFECT
```

Comment:
Almost everything beneath this is 1:1 copy paste from xenos, is this intended?

Reply:
Yes. All of those flags, as of current stage of WIP are intentionally there. Might change with balance adjustments once its past 'spriteless proof of concept' stage.